### ISSUE TITLE
Commands don't work in the content of the page option

### ISSUE THREAD

**Issue by nickw2066** [zepinglee/citeproc-lua#78]
**Describe the bug**
Commands don't work in the content of the page option.

**Additional information**
- TeX distribution: MacTeX 2024
- Package `citation-style-language` version: latest release
- LaTeX engine: `lualatex`

**To Reproduce**
```tex
\documentclass{article}

\begin{filecontents}[overwrite, noheader]{\jobname.json}
[
    {
        "id": "King2008-Gramma",
        "type": "thesis",
        "author": [
            {
                "family": "King",
                "given": "John T."
            }
        ],
        "title": "A Grammar of Dhimal",
        "publisher-place": "Netherlands",
        "publisher": "Universiteit Leiden",
        "URL": "https://scholarlypublications.universiteitleiden.nl/handle/1887/13072",
        "genre": "doctoral dissertation",
        "issued": {
            "date-parts": [
                [
                    "2008",
                    "9",
                    "3"
                ]
            ]
        },
        "accessed": {
            "date-parts": [
                [
                    "2024",
                    "08",
                    "13"
                ]
            ]
        }
    }
]
\end{filecontents}

\usepackage{citation-style-language}
\cslsetup{style = turabian-fullnote-bibliography}
\addbibresource{\jobname.json}

\newcommand{\hlm}{hlm.}

\begin{document}

Commands don't work in the content of the page option.\cite[page = {PDF \hlm\ 20}]{King2008-Gramma}

\end{document}
```

**Screenshots**
![Screenshot 2024-09-20 at 12 41 29 PM](https://github.com/user-attachments/assets/359e1e9c-d721-4ded-9de7-3f1e8f0765d3)

[Turabian CSL](https://www.zotero.org/styles/turabian-fullnote-bibliography)

**Comment by zepinglee** [comment]
This bug is fixed in 381f661.

BTW what does “hlm.” mean?

**Comment by nickw2066** [comment]
Thank you. The abbreviation "hlm." means "halaman" ('page' in Indonesian).
